dbt_coves/tasks/load/main.py

```diff
@@ -21,7 +21,7 @@ def register_parser(cls, sub_parsers, base_subparser):
         ext_subparser = sub_parsers.add_parser(
             "load",
             parents=[base_subparser],
-            help="Loads data from different systems.",
+            help="Loads configurations from Airbyte",
```

Comment:
Airbyte should not be mentioned here yet, in case we have "dbt-coves load streamsets" in the future.

Reply:
@ssassi I don't know that we should make something generic when it is not.  I think it is highly unlikely that we will have loading of streamsets.  It is more confusing to an end user IMO bec we are only doing one thing but giving an impression that we do more.

Reply:
It's a matter of being consistent, the help message of "dbt-coves" is not displaying the help text of one of its sucbommands. Same thing here, "dbt-coves load" has subcommands implemented and it should not confuse the user. What about "Loads configurations into different systems, such as Airbyte."? An alternative to this would be combining `dbt-coves load airbyte` into `dbt-coves load-airbyte` and `dbt-coves extract-airbyte`.

Reply:
ok, that works

Reply:
@BAntonellini I see you resolved this thread but can't see the solution. What was it?